sys-devel/solidity/solidity-0.4.2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+CMAKE_MIN_VERSION="3.0.0"
```

Comment:
is this check really important? I mean we still have 2.8 in the tree, but 3.5.2 is already stable...